lib/utils/getOverlayEntry.js

```diff
@@ -0,0 +1,57 @@
+const querystring = require('querystring');
+
+/**
+ * Creates an object that contains two entry arrays: the prependEntries and overlayEntries
+ * @param {import('../types').NormalizedPluginOptions} options Configuration options for this plugin.
+ * @param {import('webpack').Compiler.options.devServer | undefined} devServer The webpack devServer config
+ * @returns {import('./injectRefreshEntry').AdditionalEntries} An object that contains the Webpack entries for prepending and the overlay feature
+ */
+function getOverlayEntry(options, devServer) {
```

Comment:
You think it would be good to change this into an object (I'm not sure if we would revisit this and add more parameters in the future)?

```suggestion
function getOverlayEntry({ devServer, options }) {
```